src/ScheduleService.Validation/Group/EditGroupRequestValidator.cs

```diff
@@ -0,0 +1,10 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.ScheduleService.Models.Dto.Requests.Group;
+using LT.DigitalOffice.ScheduleService.Validation.Group.Interfaces;
+
+namespace LT.DigitalOffice.ScheduleService.Validation.Group;
+
+public class EditGroupRequestValidator : AbstractValidator<EditGroupRequest>, IEditGroupRequestValidator
```

Comment:
it should take JsonPatchDocumnet\<EditGroupRequest\>

Reply:
Instead of EditGroupRequest?

Reply:
yes and also it should take id of group you are to edit

Reply:
+

Reply:
I guess you need smth like this
![image](https://github.com/LT-Students/ScheduleService/assets/101804724/08aa6bb6-2109-400c-9bb1-0d6127207bb1)